front/src/components/Login.jsx

```diff
@@ -0,0 +1,78 @@
+export default function Login() {
```

Comment:
There is a login page on main try using this one

Reply:
I didn't understand, you mean use Login.jsx or LoginForm.jsx ?